src/main/resources/static/js/index.js

```diff
@@ -90,3 +90,123 @@ function getFilterResults() {
     );
 }
 
+function post(url, data){
+    $.ajax({
+        type: "POST",
+        url: url,
+        data: JSON.stringify(data),
+        dataType : "text",
+        contentType: "application/json",
+        success: function(data) {
+            // Update the table
+            //taken out to reload the page such that new courses are seen.
```

Comment:
Remove commented out stuff if uneeded.

Reply:
Will do, there are a few things like that going to push a commit that cleans up the code a bunch soon